.release/ci.hcl

```
@@ -0,0 +1,138 @@
+schema = "1"
+
+project "terraform" {
+  // the team key is not used by CRT currently
+  team = "terraform"
+  slack {
+    notification_channel = "C02A58BNPU6" // TODO: Placeholder
```

This is the channel where we want release notifications to appear, right?

```suggestion
    notification_channel = "C011WJ112MD" // #feed-tf-core
```